src/fava/templates/macros/_commodity_macros.html

```diff
@@ -0,0 +1,20 @@
+{% macro render_currency(currency) %}
+{% set alias = ledger.commodities.get(currency).meta.get('name') if ledger is defined else none %}
```

Comment:
The `if ledger is defined else none` part shouldn't really be necessary IMHO. If `ledger` is not defined, that sounds like an error we shouldn't silently swallow.

Reply:
Also, doesn't the `.meta.get('name')` part error for undefined currencies?